locales/en/badge.json

```
@@ -1,12 +1,12 @@
 {
   "title": {
-    "connectWallet": "You must connect wallet first",
-    "mintable": "You can mint this badge with",
-    "notMintable": "You canont mint this badge. to Get this badge you need to met certain criteria",
-    "freeMintable": "You can mint this badge free",
-    "minted": "🎉🎉🎉 You have Badge already !! "
+    "connectWallet": "You must connect to a wallet first!",
+    "mintable": "You can mint this Badge with",
+    "notMintable": "You cannot mint this Badge. To get this Badge you need to meet a certain criteria.",
+    "freeMintable": "You can mint this Badge for free.",
+    "minted": "🎉🎉🎉 You have the Badge already! 🎉🎉🎉"
```

Assuming from context that this appears when the user mints the badge, as opposed to when they cannot mint a second one.

Unsure about the wording, but maybe it should sound more like:

```suggestion
    "minted": "🎉🎉🎉 Congratulations! You minted a new badge! 🎉🎉🎉"
```

---

I think it was previously "You have Badge already!" so I assumed that this showed up when the user tried to mint another one while already possessing a badge.